lib/deface/action_view_extensions.rb

```diff
@@ -41,7 +41,7 @@ def render(view, locals, buffer=nil, &block)
 
       if @compiled && !mod.instance_methods.include?(method_name.to_sym)
         @compiled = false
-        @source = refresh(view).source
+        @source = refresh(view).source if Deface.before_rails_6?
```

Comment:
Since this line spawns a `NoMethodError`, duck typing would suggest testing for `respond_to?(:refresh)` instead of checking for the version of Rails. I think that would be more flexible.

Reply:
Thank you for your review! I'll modify it

Reply:
Ah, `if respond_to?(:refresh)` may not avoid the warning message in Rails 6.0  ( https://github.com/spree/deface/issues/204 ). 

But it's not a big deal, IMO.

Reply:
force-pushed   7bf8fdc to 3ed7fee now 🙏